yarn.lock

```diff
@@ -40,6 +96,89 @@
     jsesc "^2.5.1"
     source-map "^0.5.0"
 
+"@babel/generator@^7.17.3", "@babel/generator@^7.17.7":
```

Comment:
Interesting that we have two versions now - worth looking into?

Reply:
@jmelberg-okta  Dealing with resolutions is getting tricky. Newer versions of some of these packages now publish pure ESM modules and hence needs transformation for eslint, jest, etc and this might need more work. Unfortunately this will go into the backlog due to the limited bandwidth in our team if I create a ticket. We are exploring few alternatives for lerna, yarn 3 etc. Hopefully we can address this is in the future.